server/src/controllers/loans.controllers.ts

```diff
@@ -0,0 +1,40 @@
+import OutstandingLoan from "../models/OutstandingLoan";
+import {type Request, type Response} from "express";
+
+const { ObjectId } = require("mongoose").Types;
+
+const editLoan = async (req: Request, res: Response): Promise<any> => {
+  try {
+    const id = new ObjectId(req.body._id);
+    if (id) {
+      console.log(req.body);
+      await OutstandingLoan.findByIdAndUpdate(id, {$set: req.body.content})
```

Comment:
req.body.content and req.body._id isn't Typesafe here. Please import the interface from `OutstandingLoan.ts` and use the pattern seen here: (https://github.com/ChangePlusPlusVandy/vac-net/blob/a9c4e9bab9f8ff75490e1b4e561bcb6290a39fbc/server/src/controllers/session.controllors.ts#L7)